src/dialects/abstract/query-interface.js

```diff
@@ -638,14 +643,16 @@ export class QueryInterface {
   /**
    * Show indexes on a table
    *
-   * @param {string} tableName table name
-   * @param {object} [options]   Query options
+   * @param {TableName} tableName Table name, possibly with schema and/or delimiter
+   * @param {object}    [options] Query options
    *
    * @returns {Promise<Array>}
    * @private
    */
   async showIndex(tableName, options) {
-    const sql = this.queryGenerator.showIndexesQuery(tableName, options);
+    const table = this.queryGenerator.extractTableDetails(tableName);
```

Comment:
showIndexesQuery should call extractTableDetails, so this method doesn't need to